packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.component.js

```diff
@@ -0,0 +1,23 @@
+import controller from './configuration.controller';
+import template from './configuration.html';
+
+export default {
+  bindings: {
+    cdn: '=',
```

Comment:
Just so you know, I hate this, it means I have to go to the `*.component.js` file to know how to use the component, whereas is everything was in `'<'`, we would be so fine 😅

Reply:
It won't work well in the case of a form though 😬